WebView/app/src/main/java/com/android/samples/webviewdemo/MainActivity.kt

```diff
@@ -16,9 +16,10 @@
 
 package com.android.samples.webviewdemo
 
-import android.content.Context
+import android.annotation.SuppressLint
```

Comment:
Remove this. I think Android Studio can automatically remove unused imports and sort them for you. See https://medium.com/android-dev-hacks/android-studio-tips-tricks-part-2-advanced-3cfd23fc493e